### PR TITLE
fix: Add Ocp clusterrole

### PR DESCRIPTION
**Pull request by johnkost**
Checklist:

* [x] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA and the build is green.
* [x] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by johnkost** [comment]
I don't think I can limit this to just `analysisruns`. Will revisit this shortly and reopen PR

File: charts/argo-rollouts/templates/argo-rollouts-clusterrole.yaml [hunk 1]
```diff
@@ -106,4 +106,16 @@ rules:
   - watch
   - get
   - update
+- apiGroups:
+  - analysisruns
```
Comment (zroubalik) [inline charts/argo-rollouts/templates/argo-rollouts-clusterrole.yaml:110]: isn't this apiGroup covered by the one `"*"` below?

Comment (johnkost) [reply charts/argo-rollouts/templates/argo-rollouts-clusterrole.yaml:110]: Yes. Let me remove this

**Comment by zroubalik** [comment]
This fix should be included in rollouts repo as well, right?
https://github.com/argoproj/argo-rollouts/tree/master/manifests

**Comment by johnkost** [comment]
> This fix should be included in rollouts repo as well, right?
> https://github.com/argoproj/argo-rollouts/tree/master/manifests

Opened https://github.com/argoproj/argo-rollouts/pull/510

**Comment by cabrinha** [comment]
@johnkost please resolve conflicts here

**Pull request merged**